src/renderer/pages/player-page.js

```diff
@@ -585,6 +586,8 @@ function renderPlayerControls (state) {
 // Renders the loading bar. Shows which parts of the torrent are loaded, which
 // can be 'spongey' / non-contiguous
 function renderLoadingBar (state) {
+  if (config.IS_TEST) return // Don't integration test the loading bar. Screenshots won't match.
```

Comment:
So, is this now the preferred approach vs. creating transparent regions in the test images?